### ISSUE TITLE
[nl] unexpected uppercase suggestion

### ISSUE THREAD

**Issue by baarsrj** [languagetool-org/languagetool#8953]
![afbeelding](https://github.com/languagetool-org/languagetool/assets/8479998/11d3501c-0518-4daf-a4ac-cf73d06f96a7)

is last night's diff from

![afbeelding](https://github.com/languagetool-org/languagetool/assets/8479998/376bea4f-a516-4d3e-b81b-57709a8ed700)

Has something automagically change in the 'alllower' ?

**Comment by danielnaber** [comment]
Yes, @jaumeortola can probably tell you how to avoid the all-uppercase suggestions-

**Comment by baarsrj** [comment]
The change should not overrule an explicit xml statement to suggest allower.One solution might be to ignore suggestions with spaces in the new code. Op 15 jul. 2023 09:59 schreef Daniel Naber ***@***.***>:
Yes, @jaumeortola can probably tell you how to avoid the all-uppercase suggestions-

**Comment by jaumeortola** [comment]
To avoid all-uppercase suggestions, now you need to use a `<match>` with the attribute `case_conversion` at the start of the first suggestion (https://github.com/languagetool-org/languagetool/issues/7125#issuecomment-1637177633).

**Comment by baarsrj** [comment]
I don't understand. I explicitly command part of the suggestion to be lowercase, but that is overruled? 

Could you please edit the mentioned Dutch rule to demonstrate?

**Comment by mark-baas** [comment]
I'll modify the rule, I've done it with another to avoid unnecessary caps.

**Comment by baarsrj** [comment]
Jaume already did this.Op 17 jul. 2023 10:17 schreef mark-baas ***@***.***>:
I'll modify the rule, I've done it with another to avoid unnecessary caps.

**Comment by jaumeortola** [comment]
> I explicitly command part of the suggestion to be lowercase, but that is overruled?
> I still consider it a bug though, since en explicitly states alllower, and still turns up all upper. When a statement does not lead to the specified output, it is a bug.

I know. I could change the logic of the implementation and do this: if any part of any suggestion has a `case_conversion` attribute, then do not suggest all-uppercase replacements at all. But I'm afraid it will cause many undesired results in many languages. I will test anyway.
